### ISSUE TITLE
Item IDs absent from rendered HTML

### ISSUE THREAD

**Issue by opn** [fedwiki/wiki-server#139]
Testing with curl I find that the HTML returned from routes ending in .html do not have ids on the paragraph divs returned. This seems to be added by the javascript.

Would be good to add these to the vanilla markup so that programmes can fetch specific portions if rendered html.

**Comment by WardCunningham** [comment]
Good idea. Should be easy. But the server-side rendering into html is only for a few item types intended to be read by robots. The rendering could be easily duplicated by a consumer from the json data.

https://github.com/fedwiki/wiki-server/blob/5632501ee391256c0342913ece98bc3693159801/lib/server.coffee#L56-L76

It might be interesting to build a "rendering proxy" that would run client plugins server-side for any lineup url including those that span multiple sites. Working out verses of the 99 Bottles of Beer song would be a good test case.

**Comment by WardCunningham** [comment]
Speaking of rendering, github has done a nice job of expanding my code reference into a code window. I wonder how that comes out in watch list email? They don't send me a copy but maybe someone else can tell us how hard they try to match the rendering they do online.

For comparison, here is a screen shot of what I am seeing from me in the comment stream.

![image](https://user-images.githubusercontent.com/12127/37156382-d2478d7a-229a-11e8-9793-070d0621b0b8.png)
